controllers/csvHelpers.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+
+const readFile = (file, parser) => {
+	try {
+		const content = fs.readFileSync(file).toString('utf-8');
+		console.log('content', content);
+		return parser && typeof parser === 'function' ? parser(content) : content;
+	} catch (err) {
+		throw err;
+	}
+};
+
+const csvParser = (csv) => {
+	const csvLines = csv.split(/\r\n|\n/);
+	const headers = csvLines[0].split(',');
+	const lines = [];
+
+	for (let i = 1; i < csvLines.length; i++) {
+		const data = csvLines[i].split(',');
+
+		if (data.length == headers.length) {
+			const jsonObj = {};
+
+			for (let j = 0; j < headers.length; j++) {
+				jsonObj[headers[j]] = data[j];
+			}
+
+			lines.push(jsonObj);
+		}
+	}
+
+	return lines;
+};
+
+module.exports = {
```

Comment:
I'm not sure whether this file could be moved to `utils` folder, maybe that's a good place for it. Also please add test cases for it.